src/fairchem/core/trainers/base_trainer.py

```diff
@@ -545,10 +546,9 @@ def load_model(self) -> None:
                 )
             self.logger.log_summary({"num_params": num_params})
 
-        if distutils.initialized() and not self.config["noddp"]:
+        if distutils.initialized():
```

Comment:
Is there a case when distutils is not initialized here and an error has not occured?

Reply:
i think there are some tests where they do predict and not have DDP initialized so i kept it here